KeyDistribution_BB84/Tasks.qs

```diff
@@ -129,14 +181,19 @@ namespace Quantum.Kata.KeyDistribution {
     }
 
 
-    // Task 2.5. Was communication secure?
+    // Task 2.5. Check if error rate was low enough
+    // The main trace evesdroppers can leave on a key exchange is to introduce
+    // more errors into the transmission. Alice and Bob have already characterized
```

Comment:
At the first read, "have already characterized" sounds like this is something they've done in the kata, and I start trying to remember whether there was an exercise about this. How about this?

```suggestion
    // more errors into the transmission. Alice and Bob should have characterized
```